custom/icds_reports/ucr/reports/mobile_mpr_7_growth_monitored.json

```diff
@@ -108,6 +103,16 @@
         "field": "last_date_gmp",
         "format": "%Y-%m"
       },
+      {
+        "display": "Age Group",
+        "column_id": "age_group",
+        "type": "conditional_aggregation",
+        "whens": {
+            "0 <= age_in_months AND age_in_months < 12": "less_than_1",
```

Comment:
Any reason there's a switch from months to years here? Reports get really confusing when there are switches in vocabulary units random places in the pipeline.